lib/maze/client/selenium/bs_client.rb

```diff
@@ -11,7 +13,7 @@ def start_session
           browser = browsers[config.browser]
 
           if config.legacy_driver?
-            capabilities = Selenium::WebDriver::Remote::Capabilities.new
+            capabilities = ::Selenium::WebDriver::Remote::Capabilities.new
```

Comment:
This was the main issue.  Because the line is in a module called `Something::Something::Selenium`, by default Ruby thinks I want want a namespace relative to the current one.  The `::` at the start makes it an absolute one.